README.md

```diff
@@ -62,6 +62,14 @@ We use [Discord](https://discordapp.com/) for community discussion. You can use
 
 Notion is intended to compile with all versions of Rust newer than 1.20.
 
+## Tests
+
+To run the tests in this repo, run the following:
+
+```
+cargo test --features mock-network
```

Comment:
This should be `cargo test --all --features mock-network`, to include all the unit tests from the sub-crates in this project

Reply:
Sorry about that, thank you for pointing out the correct command!